tests/test_views.py

```diff
@@ -286,6 +286,14 @@ def test_invalid_groupmembership(self, user_client):
         response = user_client.get(self._get_url(1))
         assert response.status_code == HTTPStatus.NOT_FOUND
 
+    def test_non_manager_cannot_access(self, no_manager_group, auth_client_factory):
+        """Test that a user who is not a manager cannot access the view."""
+        non_manager, group = no_manager_group
+        client = auth_client_factory(non_manager)
+        response = client.get(self._get_url())
+        assert response.status_code == HTTPStatus.FORBIDDEN
+        assert response.content == b"Permission denied"
+
```

Comment:
This is checking the same thing as `test_not_group_owner` (which could be renamed `test_not_group_owner_or_manager`). We need a test to show that managers _can_ access the view.